rl_loop.py

```diff
@@ -49,20 +52,36 @@ def print_flags():
                     for flag, value in flags.items()))
 
 
+def get_models():
+    all_models = gfile.Glob(os.path.join(MODELS_DIR, '*.meta'))
+    model_filenames = [os.path.basename(m) for m in all_models]
+    return [(shipname.detect_model_num(m), shipname.detect_model_name(m))
+            for m in model_filenames]
+
+
 def get_latest_model():
     """Finds the latest model, returning its model number and name
 
     Returns: (17, 000017-modelname)
     """
-    all_models = gfile.Glob(os.path.join(MODELS_DIR, '*.meta'))
-    model_filenames = [os.path.basename(m) for m in all_models]
-    model_numbers_names = [
-        (shipname.detect_model_num(m), shipname.detect_model_name(m))
-        for m in model_filenames]
+    model_numbers_names = get_models()
     latest_model = sorted(model_numbers_names, reverse=True)[0]
     return latest_model
 
 
+def enough_games_to_train():
+    model_numbers_names = get_models()
+    if len(model_numbers_names) <= 2:
```

Comment:
if the number of models is less then 2, then we always have enough models to train? This is a little unintuitive, so a comment would be helpful

Reply:
there's an edge case if you only have the first model.  So either you bootstrap, and let your cluster spin for a little bit, or you just shrug and go ahead and train the first model with too little data and let it catch up for all models > 2

Reply:
(comment added)